### DISCUSSION TITLE
Author and Editor Role has no access to collections

### DISCUSSION THREAD

**Discussion by NicoHood** [strapi/strapi#7488] [Legacy Topics]
**Describe the bug**
Hi,
I wanted to try out the roles and permission feature of the strapi community edition. But that does not seem to work.

**Steps to reproduce the behavior**

1. Add a new User
2. Set his role to Author (or Editor)
3. Login with this user
4. The user does not see any collection, but he can upload media. In editor mode he also sees media from other users.

**Expected behavior**
The user should at least see the available collections as well.

**System**

- Node.js version: 14.8.0
- NPM version: ---
- Strapi version: 3.1.4
- Database: sqlite
- Operating system: archlinux

**Additional context**
Beside this issue I want to point out, that the plugin with Roles and Permissions (dont confuse with the settings section) is a bit confusing to me. Why do we have this, if we already have user settings? That does not make sense to me, maybe someone can explain me that as well. Do I need to set additional public or authenticated settings to allow my strapi users to read the collections? (I guess not, it seems that those settings are for the API itself, correct?)

**Comment by derrickmehaffy** [comment]
Hi @NicoHood 

The default action when a new content-type is added is to not add permissions to the the Author or Editor roles (as a side point we will likely be renaming the `users-permissions` plugin to make it a bit easier to understand the difference between it and the Admin RBAC or Role Based Access Control).

If you navigate into the Settings => Roles => Author/Editor => Collection-Types, you can select the models they have access to, and expanding those you can control the fields (just not the CRUD actions as that is enterprise only).

![image](https://user-images.githubusercontent.com/8593673/90407683-9da98600-e05b-11ea-9475-eae9a00e77fc.png)


You are correct that the users-permissions plugin here doesn't matter as that is only for consuming the generated API routes in use by your "frontend". I will also be moving this over to a discussion as this isn't really a bug but more of a question.

Let me know if this answers your question :)
